zipline/data/minute_bars.py

```diff
@@ -465,6 +489,17 @@ def __init__(self,
     def first_trading_day(self):
         return self._start_session
 
+    def ohlc_ratio_for_sid(self, sid):
+        if self._ohlc_ratios_per_sid is not None:
+            try:
+                return self._ohlc_ratios_per_sid[sid]
+            except KeyError:
```

Comment:
If the `_default_ohlc_ratio` is being used as a fallback/default, should it be returned here?